wasp/fonts/sans18.py

```diff
@@ -0,0 +1,264 @@
+# Code generated by font_to_py.py.
```

Comment:
Wouldn't this be better in a separate PR?

Reply:
You are absolutely correct, this was intended to be a separate PR.
I have no idea how I messed this up

Reply:
If you'd run these commands it should fix it:

```sh
git rebase -i HEAD^^
```
And write "drop" next to the font commit.
After that, create a new branch and run this command:
```sh
git cherry-pick 919f8b86de5704a3749eb565a83b8c0b8d9d1d1d
```
And submit a separate PR for it.

Reply:
Make sure that your new branch is based on **this** master, not the one you use - otherwise, your font PR will have both the font and the battery level changes.

Generally, it's easier to work with a master branch that's synced to this branch, and each time you want to add a feature branch from it and add your feature there.

Reply:
Always glad to help :)